.github/workflows/instrumented-tests.sh

```
@@ -0,0 +1,35 @@
+
+API_LEVEL=$1
+SKIN=$2
+
+TARGET=""
+if [ $API_LEVEL -ge 28 ]; then
+    TARGET="google_apis_playstore"
+else
+    TARGET="default"
+fi
+
+PACKAGE="'system-images;android-$API_LEVEL;$TARGET;x86_64'"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Rewrite using set/"$@" or functions.

(SC2089)